### ISSUE TITLE
[Feature : Tracker implementation] Hard to understand item codes

### ISSUE THREAD

**Issue by benoittachet** [DodoBirby/MTTNE-Randomiser#1]
Hello DodoBirby,

I'm trying to create an emotracker pack for your randomizer, and I'm trying to understand the item codes that are in the "requirements.txt" file. According to what I understood, the first part of the codes corresponds to the column of the room (and not even every time), but I'm quite stuck with the second part, which I suppose is for the line (but I'm not sure at all)
Until now, I've been able to find which was which wxploring the map, but I'm still curious about it (if you're the one who made these codes, of course).

Thank you for your answer !

**Comment by DodoBirby** [comment]
Yeah don't worry about not understanding the item codes since they aren't really meant to be easily readable, they represent the offset of the item in the map file so 0xc44 (long beam location) is c44 (in hexadecimal) bytes from the start of the map file.

I'll commit the notes I made when getting the item locations in a file called "Offset List"

**Comment by DodoBirby** [comment]
also apologies for the bad documentation this is one of my first "real" projects and also a first randomiser. I really didn't expect anyone to actually make anything for it.

**Comment by benoittachet** [comment]
So that's what these values are ! Okay, I get it.

No problem for the documentation, you'll learn how to do it bit by bit ^^

Great, now I can continue my tracker, it is going to be easier.

**Comment by benoittachet** [comment]
I open that one back just for one point : What does the "N6 1e" code means ? I have understood that the "1e" part was for having one ore more energy tanks, but is it for a specific number of energy tanks ?
The question is also the same for "Nb 23", "N2 22", N3 22, N1 1e, N6 24, etc.
The one that fits less there is Nb23 : 11 bombs (morph or power) needed ? That's quite surprising...

PS : Can I contact you via Discord directly if I have more questions ? I think I have found your account on Xander's server...

**Comment by DodoBirby** [comment]
The "N" means that you have at least that many items. 1e is the code for E-tanks so "N6 1e" means at least 6 E-Tanks. 23 is the code for keycards so you need at least 11 keycards to access that check (the reason so many keycards are required is because you could possibly waste keycards on the wrong door). The other ones are "22" for power bombs and "24" for gears.

To answer the PS, yes you can contact me on Discord and I am on the Xander server.
